Heroku/config/log4j2.xml

```diff
@@ -19,7 +19,7 @@
 		</Property>
 		<!-- Whitespace within color configuration (highlight/style) will be ignored -->
 		<Property name="pattern_styled">%highlight{%date{ISO8601} %-5level [%thread] %logger(%file:%line)}{${
-		pattern_level_colors}} %style{%message}{bright white}%n%style{%throwable}{BG_red,white}
+		pattern_level_colors}} %style{%message}{bright_white}%n%style{%throwable}{BG_red,white}

```

Comment:
problem introduced in https://github.com/TWiStErRob/net.twisterrob.cinema/pull/161

Reply:
https://issues.apache.org/jira/browse/LOG4J2-3561